src/app/globals.css

```
@@ -292,4 +292,16 @@ body {
 
 .text-red-600 {
   color: #dc2626 !important;
+}
+
+/* Hide cursor ONLY in Hero section */
+`@media` (pointer: fine) {
```

Stylelint (17.6.0)

[error] Unexpected unknown type selector "`@media`" (selector-type-no-unknown)

(selector-type-no-unknown)